dev-assets/index.html

```diff
@@ -63,24 +63,33 @@
     // register react component as custom element
     const WRC = WebReactComponents;
     WRC.register(MyComponent, 'my-component', [
-      // these will be json parsed
-      'name',
-      'age',
-      'fruits',
-      'config',
-      // specify a boolean attribute like this
-      '!!isActive',
-      // these will be handlers
-      'onClickButton()',
-      'onHoverName()',
-    ]);
+        // these will be json parsed
+        'name',
+        'age',
+        'fruits',
+        'config',
+        // specify a boolean attribute like this
+        '!!isActive',
+      ],
+      // specify mappers for events, the function receives the arguments
+      // the react handler receives and dispatched an event on the web component
+      // if null is returned nothing no DOM event is dispatched
+      {
+        onClickButton: e => {
+          return null; // don't dispatch more events because click is already bubbling
+        },
+        onHoverName: e => {
+          return new Event('hovername', { bubbles: true });
+        },
+      }
+    );
 
     // add some DOM Level 3 event handlers
     const c = document.getElementById('myComponent');
-    c.addEventListener('onClickButton', function(e) {
+    c.addEventListener('clickbutton', function(e) {
```

Comment:
That should be `click` no?

Reply:
Well for this example let's assume that the component has a onClickButton() and a onClickIcon() property. Then you would have to call it buttonclick to disambiguate.

Reply:
Yeah, but then it's not a good example for the test page, because it's not working ;)